tests/api.c

```diff
@@ -27091,14 +27087,14 @@ static void test_wolfSSL_ERR_print_errors(void)
     AssertNotNull(bio = BIO_new(BIO_s_mem()));
     ERR_clear_error(); /* clear out any error nodes */
     ERR_put_error(0,SYS_F_ACCEPT, -173, "ssl.c", 0);
-    ERR_put_error(0,SYS_F_BIND, -275, "asn.c", 100);
+    ERR_put_error(0,SYS_F_BIND, -295, "asn.c", 100);
```

Comment:
Why did this change from -275 to -295? Same below at 27097.

Reply:
@dgarske, We need to test for an unknown error number. Since -275 is now used, I had to choose something else. It can be any number between -276 to -300.

Reply:
Why not use `WC_LAST_E + `?. Please document with inline comment.